scripts/uberenv/project.json

```diff
@@ -1,8 +1,10 @@
 {
 "package_name" : "conduit",
-"uberenv_package_name" : "uberenv-conduit",
-"spack_url": "https://github.com/alpine-DAV/spack",
-"spack_branch": "task/2019_10_update_conduit",
+"package_version" : "master",
+"package_final_phase" : "configure",
+"package_source_dir" : "../..",
+"spack_url": "https://github.com/spack/spack",
+"spack_branch": "develop",
```

Comment:
i know we need to use a newer spack, but I am a bit worry about doing this without testing in a few places (NERSC)

I would also want it to be pinned to some version to avoid random failures when spack develop moves on.

Reply:
OK

Reply:
I can create a fork in the alpine space that points to a newer spack

Reply:
I am

Reply:
v0.13.4 is in fact too old (does not include latest conduit).
So I will try using the merge commit of this branch into develop. Then we can either keep the commit in the project.json, or use a clone is ascent.